changes.go

```diff
@@ -223,17 +223,40 @@ type TopicInput struct {
 	Topic string `json:"topic,omitempty"`
 }
 
+type Account struct {
+	AccountId int      `json:"_account_id"`
+	Name      string   `json:"name"`
+	Email     string   `json:"email"`
+	Username  string   `json:"username"`
+	Tags      []string `json:"tags,omitempty"`
+}
```

Comment:
This is not needed, as we need an `AccountInfo` type

Reply:
In a new commit I have modified